modules/appnexusAstBidAdapter.js

```diff
@@ -27,364 +24,349 @@ const NATIVE_MAPPING = {
-            // e.g., tag specifies `body` but /ut only knows `description`
-            // mapping may be in form {tag: '<server name>'} or
-            // {tag: {serverName: '<server name>', serverParams: {...}}}
-            Object.keys(bid.nativeParams).forEach(key => {
-              // check if one of the <server name> forms is used, otherwise
-              // a mapping wasn't specified so pass the key straight through
-              const requestKey =
-                (NATIVE_MAPPING[key] && NATIVE_MAPPING[key].serverName) ||
-                NATIVE_MAPPING[key] ||
-                key;
-
-              // if the mapping for this identifier specifies required server
-              // params via the `serverParams` object, merge that in
-              const params = Object.assign({},
-                NATIVE_MAPPING[key] && NATIVE_MAPPING[key].serverParams,
-                bid.nativeParams[key]
-              );
-
-              nativeRequest[requestKey] = params;
-            });
-
-            tag.native = {layouts: [nativeRequest]};
-          }
-        }
-
-        if (bid.mediaType === 'video') { tag.require_asset_url = true; }
-        if (bid.params.video) {
-          tag.video = {};
-          // place any valid video params on the tag
-          Object.keys(bid.params.video)
-            .filter(param => VIDEO_TARGETING.includes(param))
-            .forEach(param => tag.video[param] = bid.params.video[param]);
-        }
+    if (userObjBid) {
+      userObj = {};
+      Object.keys(userObjBid.params.user)
+        .filter(param => USER_PARAMS.includes(param))
+        .forEach(param => userObj[param] = userObjBid.params.user[param]);
+    }
 
-        if (bid.params.user) {
-          userObj = {};
-          Object.keys(bid.params.user)
-            .filter(param => USER_PARAMS.includes(param))
-            .forEach(param => userObj[param] = bid.params.user[param]);
-        }
+    const memberIdBid = bidRequests.find(hasMemberId);
+    const member = memberIdBid ? parseInt(memberIdBid.params.member, 10) : 0;
 
-        return tag;
-      });
+    const payload = {
+      tags: [...tags],
+      user: userObj,
+      sdk: {
+        source: SOURCE,
+        version: '$prebid.version$'
+      }
+    };
+    if (member > 0) {
+      payload.member_id = member;
+    }
+    const payloadString = JSON.stringify(payload);
+    return {
+      method: 'POST',
+      url: URL,
+      data: payloadString,
+    };
+  },
 
-    if (!utils.isEmpty(tags)) {
-      const payloadJson = {
-        tags: [...tags],
-        user: userObj,
-        sdk: {
-          source: SOURCE,
-          version: '$prebid.version$'
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function(serverResponse) {
+    const parsed = JSON.parse(serverResponse);
```

Comment:
suppose the base adapter could do the parse, unless we need custom error handling in the adapter (I don't see any here).